jenkins-ci/process.py

```diff
@@ -0,0 +1,64 @@
+
+import sys
+import os
+import argparse
+import lib.bisect_lib as cb
+# from lib import common_lib as cb
+def fetch(sid,type):
```

Comment:
Add a whiteline here and doc string for this function. Also please run pycodestyle against this and other py files(that you have changed) and fix formatting issues (if any)